OloEngine/src/OloEngine/Renderer/Commands/FrameResourceManager.cpp

```
@@ -292,18 +292,18 @@ namespace OloEngine
-        {
-            OLO_CORE_WARN("FrameResourceManager::WaitForFence: Fence wait timed out!");
-            return false;
-        }
-        else if (result == GL_WAIT_FAILED)
+        constexpr u64 TIMEOUT_NS = 1000000000ULL; // 1 second
+        switch (RenderCommand::ClientWaitFence(fenceId, TIMEOUT_NS))
         {
-            OLO_CORE_ERROR("FrameResourceManager::WaitForFence: Fence wait failed!");
-            return false;
+            case RHI::FenceStatus::TimeoutExpired:
+                OLO_CORE_WARN("FrameResourceManager::WaitForFence: Fence wait timed out!");
```

Cppcheck (2.21.0)

[style] The function 'SetViewState' is never used.

(unusedFunction)